extensions/wikia/CommunityPage/CommunityPage.i18n.php

```diff
@@ -113,13 +114,14 @@
 	'communitypage-modal-tab-all' => 'All members modal tab',
 	'communitypage-modal-tab-admins' => 'Admins modal tab',
 	'communitypage-modal-tab-all-contribution-header' => 'A message shown in a header of the Contributors modal over dates of last contributions of users.',
-	'communitypage-modal-tab-loadingerror' => "Error message shown when ajax call for data failed",
+	'communitypage-modal-tab-loadingerror' => 'Error message shown when ajax call for data failed',
 	'communitypage-no-contributions' => 'Text indicating that there are no contributions yet this week',
 	'communitypage-no-admins' => 'Text indicating that the wiki has no admins.',
 	'communitypage-no-admins-contact' => 'Contact us to become the first admin text',
 	'communitypage-communitycentral-link' => 'Link to the community page for this language',
 	'communitypage-no-members' => 'Text for zero state of all members list',
-	'communitypage-help-us-grow' => 'Entry point for logged in users. Invitation to help this community',
+	'communitypage-help-us-grow-prefix' => 'Entry point for logged in users. Invitation to help this community, first part of text',
+	'communitypage-help-us-grow' => 'Second part of text, including sitename',
```

Comment:
I think just for clear description for translators we could repeat description of it's purpose to second part as well, e.g.
'communitypage-help-us-grow' => 'Entry point for logged in users. Invitation to help this community, second part of text, including sitename',

Also I'd recommend tie these names more explicit, like:
communitypage-help-us-grow-before-sitename
communitypage-help-us-grow-after-sitename-included
what do you think